winbuild/build_prepare.py

```diff
@@ -569,7 +569,11 @@ def build_pillow():
         *prefs["header"],
         cmd_set("DISTUTILS_USE_SDK", "1"),  # use same compiler to build Pillow
         cmd_set("py_vcruntime_redist", "true"),  # always use /MD, never /MT
-        r'"{python_dir}\{python_exe}" setup.py build_ext --vendor-raqm --vendor-fribidi %*',  # noqa: E501
+        r'"{python_dir}\{python_exe}" -m pip install --upgrade pip',
+        r'"{python_dir}\{python_exe}" -m pip install . '
+        r'--global-option="--vendor-raqm" '
+        r'--global-option="--vendor-fribidi" '
+        r'--global-option="%*"',
```

Comment:
A quick and dirty solution to get this unblocked would be to instead require the calling script to wrap arguments using `--global-option=...`.
```suggestion
        r'%*',
```

Reply:
Ah yes, updated!

It's still not building the wheel, will have to check that later...

Reply:
You mean for the PR? Wheel building is disabled on PRs, but a wheel was built for the push in your repo: https://github.com/hugovk/Pillow/actions/runs/5105935733/jobs/9177831445

Reply:
Aha! There we go!